modules/common/affinity/affinity_test.go

```diff
@@ -51,9 +51,7 @@ func TestDistributePods(t *testing.T) {
 
 	t.Run("Default pod distribution", func(t *testing.T) {
 		g := NewWithT(t)
-
-		d := DistributePods("ThisSelector", []string{"selectorValue1", "selectorValue2"}, "ThisTopologyKey")
-
+		d, _ := DistributePods("ThisSelector", []string{"selectorValue1", "selectorValue2"}, "ThisTopologyKey", nil)
```

Comment:
can we add another test case to use the override? other then that this looks good to me. lets get also feedback from @olliewalsh

Reply:
thanks!